.circleci/config.yml

```diff
@@ -62,92 +57,14 @@ commands:
           name: Pip freeze
           command: pip freeze
 
-  setup_pre_commit:
-    description: Install pre-commit hooks
-    steps:
-      - run:
-          name: Install pre-commit hooks
-          command: pre-commit install --install-hooks
-      - run:
-          name: Run pre-commit hooks
-          command: pre-commit install --hook-type pre-push
```

Comment:
I removed this `pre-push` bit since we don't have any such hooks. Everything else from this removed code chunk is still there, just moved to `jobs`.